### PR TITLE
Add local flag to force uploading to local frontend

### PR DESCRIPTION
**Pull request by markvincze**
With this PR if we do `appix push --local` it will upload to the rwd instance running on localhost instead of the one returned by the catalog (which is usually `https://fireball-dev.development.travix.com/`).

### CODE REVIEW

File: mybuild.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+go build -o ~/.appix/appix -i .
```
Comment (alexmiranda) [inline mybuild.sh]: is this file really necessary?

Comment (markvincze) [inline mybuild.sh]: Oops, I didn't want to commit this, I'll remove.

**Comment by alexmiranda** [comment]
👍 LGTM
